import_export/src/file/export.rs

```diff
@@ -27,6 +30,32 @@ pub enum ExportError {
 
 type Result<T, E = ExportError> = std::result::Result<T, E>;
 
+/// Represents a parquet file that is being exported
+#[derive(Debug)]
+pub struct ExportedParquetFileInfo {
+    /// the file is index/num_parquet_files
+    pub index: usize,
```

Comment:
What? I'm having a bit of trouble parsing the sentence. I THINK you mean something like:

```suggestion
    /// Unique index of this file for this particular export job.
    ///
    /// Indices run from 0 (inclusive) to [`num_parquet_files`](Self::num_parquet_files) (exclusive).
    pub index: usize,
```